modules/playercards/metadata/FightPropRule.json

```diff
@@ -403,5 +403,14 @@
     "暴击伤害",
     "元素精通",
     "元素充能效率"
-  ]
-}
\ No newline at end of file
+  ],
+  "瑶瑶": [
+    "攻击力百分比",
+    "暴击率",
+    "暴击伤害",
+    "元素精通",
+    "元素充能效率",
+    "生命值百分比",
+    "治疗加成"
+  ],
+}
```

Comment:
modules/playercards/metadata/FightPropRule.json#L416
Unexpected character ('}' (code 125)): was expecting double-quote to start field name